pkg/syncer/metadatasyncer.go

```diff
@@ -2917,3 +2922,151 @@ func startStoragePolicyQuotaCRInformer(ctx context.Context, cfg *restclient.Conf
 	}()
 	return nil
 }
+
+// storagePolicyUsageCRSync patches StoragePolicyUsage CRs for k8s PVs in Bound state
```

Comment:
Update the comment to include "creates".

Reply:
Done